core/serial_command.py

```diff
@@ -362,30 +315,30 @@ def fire_continuous(self):
         self.disable_external_trigger()
         if self._firing is True:
             raise tellie_exception.TellieException("Cannot fire, already in firing mode")
-        self._send_command(_cmd_fire_continuous, False)
+        self._send_command(p._cmd_fire_continuous, False)
         self._firing = True
         self._force_setting = False
 
-    def read_buffer(self, n=100):
+    def read_buffer(self, n=p._read_bytes):
         return self._serial.read(n)
 
     def stop(self):
         """Stop firing tellie"""
         self.logger.debug("Stop firing!")
         #Disable external trigger before we do anything
         self.disable_external_trigger()
-        self._send_command(_cmd_stop, False)
-        buffer_contents = self._serial.read(100)
+        self._send_command(p._cmd_stop, False)
+        buffer_contents = self._serial.read(p._read_bytes)
```

Comment:
here's an example of what I've mentioned above. We should use the read buffer method instead !